src/Desktop.Analyzers/Desktop.Analyzers.md

```diff
@@ -48,6 +48,16 @@ Severity: Warning
 
 Help: [http://msdn.microsoft.com/library/ms182350.aspx](http://msdn.microsoft.com/library/ms182350.aspx)
 
+### CA3075: Insecure DTD processing in XML ###
+
+Using XmlTextReader.Load(), creating an insecure XmlReaderSettings instance when invoking XmlReader.Create(), setting the InnerXml property of the XmlDocument and enabling DTD processing using XmlUrlResolver insecurely can lead to information disclosure. Replace it with a call to the Load() method overload that takes an XmlReader instance, use XmlReader.Create() to accept XmlReaderSettings arguments or consider explicitly setting secure values. The DataViewSettingCollectionString property of DataViewManager should always be assigned from a trusted source, the DtdProcessing property should be set to false, and the XmlResolver property should be changed to XmlSecureResolver or null.�
```

Comment:
`XmlTextReader` doesn't have a `Load` method (or at least, I couldn't find it documented). Did you mean `.Read`?